cli/tests/pcluster/templates/test_shared_storage.py

```diff
@@ -202,6 +203,21 @@ def assert_sg_rule(
     assert_that(sg_rules).is_length(1)
 
 
+def test_non_happy_ontap_and_openfsx_mounting(mocker, test_datadir):
+    dummy_api = _DummyAWSApi()
+    dummy_api._fsx.set_non_happy(True)
```

Comment:
We want to be more specific here: how can I understand from `_fsx.set_non_happy(True)` that you're injecting a faulty behaviour for the describe_volume API? It should rather be something more specific like `mock_describe_volume_failure(True, AWSClientError)`

Reply:
Yes, I agree. I'll rename it.